python/paddle/fluid/layers/control_flow.py

```diff
@@ -941,9 +941,9 @@ def array_read(array, i):
     Examples:
         .. code-block:: python
 
-          tmp = fluid.layers.zeros(shape=[10], dtype='int32')
+          array = fluid.layers.create_array(dtype='float32')
```

Comment:
is there any test to cover this issue?

Reply:
no test as this is comment code for doc， will show in document